src/DotNetWorker.Core/FunctionsApplication.cs

```diff
@@ -64,5 +91,19 @@ public Task InvokeFunctionAsync(FunctionContext context)
                 return _functionExecutionDelegate(context);
             }
         }
+
+        public void CancelInvocation(string invocationId)
+        {
+            var invocationDetails = _functionInvocationManager.TryGetInvocationDetails(invocationId);
+            if (invocationDetails?.CancellationTokenSource is not null)
+            {
+                invocationDetails.CancellationTokenSource.Cancel();
```

Comment:
I think you should do a `TryCancel` here... if there's a race it won't throw. I *think* `Cancel` will throw if it's already canceled.

Reply:
Looks like there's no `TryCancel` or equivalent on CTS. I can wrap this in a try/catch unless there's a better approach here